Sources/Core/Public/GTMSessionFetcher/GTMSessionFetcher.h

```diff
@@ -1084,9 +1085,12 @@ __deprecated_msg("implement GTMSessionFetcherAuthorizer instead")
 @property(atomic, readonly, getter=isFetching) BOOL fetching;
 
 // Cancel the fetch of the request that's currently in progress.  The completion handler
-// will not be called.
+// will be called if the property `stopFetchingTriggersCompletionHandler` is `YES`.
```

Comment:
Perhaps, it should only be supported on the service, since it doesn't really make sense to change once the fetcher has been created. WDYT?

Reply:
Actually, that might simplify things since it locks in the state so you don't need the synchronization, etc. Would that work for the Firebase usage?

Reply:
Actually no - since we want to be able to call a combination of new and old APIs.  However, it does seem reasonable to make `stopFetchingTriggersCompletionHandler` a readonly property and I did that in the latest commit.